frame/composable-traits/src/staking/mod.rs

```diff
@@ -150,6 +150,26 @@ pub struct Stake<RewardPoolId, Balance, Reductions> {
 	pub lock: Lock,
 }
 
+pub trait ManageStaking {
+	type AccountId;
+	type AssetId;
+	type BlockNumber;
+	type Balance;
+	type RewardConfigsLimit;
+	type StakingDurationPresetsLimit;
+	type RewardPoolId;
+
+	fn create_staking_pool(
+		pool_config: RewardPoolConfiguration<
+			Self::AccountId,
+			Self::AssetId,
+			Self::BlockNumber,
+			RewardConfigs<Self::AssetId, Self::Balance, Self::RewardConfigsLimit>,
+			StakingDurationToRewardsMultiplierConfig<Self::StakingDurationPresetsLimit>,
+		>,
+	) -> Result<Self::RewardPoolId, DispatchError>;
+}
+
```

Comment:
Some docs on this trait and method would be nice, I am currently not quite sure what it does